### DISCUSSION TITLE
Integrating 3rd party SDK

### DISCUSSION THREAD

**Discussion by anto-rupak** [ionic-team/capacitor#4973] [General]
Hello Team,

Is there any way where I can Integrate the 3rd party Android SDK(Agora android) in to custom plugin in capacitor and then use it in my Ionic project? Please let me know if there is any way I can do it.

Thanks 
Anto

**Comment by jcesarmobile** [comment]
this was answered on a similar issue 
https://github.com/ionic-team/capacitor/issues/4975#issuecomment-907659453
